0.data_analysis_and_processing/mic_analysis/nbconverted/well_sirna_mic_analysis.py

```diff
@@ -234,11 +229,12 @@
 # In[17]:
 
 
-for stain in speckle_stains:
+stains = {"DAPI", "GOLD", "A647"}
+
+for stain in stains:
 
-    pattern = f"DAPI|{re.escape(stain)}"
-    stain_micdfs = micdfs.loc[micdfs["Metadata_Stain__posthoc_group0"].str.contains(pattern, regex=True) & micdfs["Metadata_Stain__posthoc_group1"].str.contains(pattern, regex=True)]
-    print(stain_micdfs)
+    comparison_stains = list(stains - {stain})
+    stain_micdfs = micdfs.loc[~(micdfs["Metadata_Stain__posthoc_group0"].str.contains(stain, regex=True) | micdfs["Metadata_Stain__posthoc_group1"].str.contains(stain, regex=True))]
```

Comment:
consider changing naming here for increased readability. stain vs stains is hard for me to differentiate. Consider maybe using `list_of_stains`